src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -78,13 +80,16 @@ private Configuration() {}
 
   private static String getPropertyFromFile(String propertyName) {
     return "null".equals(String.valueOf(readConfiguration().get(propertyName))) ? null
-        : String.valueOf(readConfiguration().get(propertyName));
+                                                                                : String.valueOf(
+                                                                                    readConfiguration()
+                                                                                        .get(
+                                                                                            propertyName));
```

Comment:
Lot of reformatting lines. Did you imported right schema into InteliJ? You can move `? null` here and in other places to new line, because this formatting will give me nightmares :x